openshift/template-post-deploy-job.yaml

```diff
@@ -10,7 +11,7 @@ objects:
 - apiVersion: batch/v1
   kind: Job
   metadata:
-    name: assisted-service-post-deploy-${IMAGE_TAG}
+    name: assisted-service-post-deploy-${JOBID}
```

Comment:
Better to use generateName?

Reply:
We need to know the name before applying the manifest... generateName is not supported by our automation.